packages/fx-core/src/component/generator/constant.ts

```diff
@@ -10,7 +10,8 @@ export const defaultTimeoutInMs = 20000;
 export const sampleDefaultTimeoutInMs = 40000;
 export const templateFileExt = ".tpl";
 export const templateAlphaVersion = "templates-0.0.0-alpha";
```

Comment:
I am not very sure about this, just double confirm, this is templates-0.0.0-alpha  not   templates@0.0.0-alpha

Reply:
Yes, it's templates-0.0.0-alpha.